src/Generators/ControllerGenerator.php

```diff
@@ -221,7 +221,7 @@ private function modelForContext(string $context)
         }
 
         $matches = array_filter(array_keys($this->models), function ($key) use ($context) {
-            return Str::endsWith($key, '/'.Str::studly($context));
+            return Str::endsWith(preg_replace('/.+\\\\/', '', $key), Str::studly($context));
```

Comment:
Not sure why this was changed, it's not really necessary for the pr.

Reply:
Okay. I think I'll revert it.